src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php

```diff
@@ -209,13 +210,17 @@ public function getNotCalledListeners(Request $request = null): array
         }
 
         $notCalled = [];
+
         foreach ($allListeners as $eventName => $listeners) {
             foreach ($listeners as $listener) {
+                $priority = $listener['priority'] ?? null;
+                $listener = $listener['listener'];
+
                 if (!\in_array($listener, $calledListeners, true)) {
                     if (!$listener instanceof WrappedListener) {
                         $listener = new WrappedListener($listener, null, $this->stopwatch, $this);
                     }
-                    $notCalled[] = $listener->getInfo($eventName);
+                    $notCalled[] = $listener->getInfo($eventName, $priority);
```

Comment:
Let's not change getInfo += the priority instead

Reply:
I mean: it's strange to add a priority argument to a getInfo method. Let's find a way that makes more sense.